bin/rad_bin.rb

```diff
@@ -1,3 +1,4 @@
+require 'English'
```

Comment:
rubocop added this when I ran it with the auto correct flag (-A) - not sure why, but will research

Reply:
seems fine, see [here](https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Style/SpecialGlobalVars)